eppo-sdk-test/helpers/AssignmentLogDataTest.cs

```diff
@@ -4,17 +4,30 @@ namespace eppo_sdk_test.helpers;
 
 public class AssignmentLogDataTest
 {
-    [Test]
-    public void ShouldBuildExperimentKey()
+    private AssignmentLogData assignmentLogData;
+
+    [SetUp]
+    public void Setup()
     {
-        var assignmentLogData = new AssignmentLogData(
+        assignmentLogData = new AssignmentLogData(
             "feature-flag",
             "allocation",
             "variation",
             "subject",
             new Subject(),
             new Dictionary<string, string>(),
             new Dictionary<string, string>());
+    }
+
+    [Test]
+    public void ShouldBuildExperimentKey()
+    {
         Assert.That(assignmentLogData.Experiment, Is.EqualTo("feature-flag-allocation"));
     }
+
+    [Test]
+    public void ShouldSetAssignmentTimestamp()
+    {
+        Assert.That(assignmentLogData.Timestamp, Is.EqualTo(DateTime.UtcNow).Within(1).Milliseconds);
```

Comment:
Slick!

Reply:
may be _too_fragile. Is 1 ms a healthy amount of buffer? should it be 5? 20?